2025-03-29 17:20:16,020 INFO    MainThread:42864 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-29 17:20:16,021 INFO    MainThread:42864 [wandb_setup.py:_flush():67] Configure stats pid to 42864
2025-03-29 17:20:16,021 INFO    MainThread:42864 [wandb_setup.py:_flush():67] Loading settings from /Users/liuyihao/.config/wandb/settings
2025-03-29 17:20:16,021 INFO    MainThread:42864 [wandb_setup.py:_flush():67] Loading settings from /Users/liuyihao/Desktop/m2cw/wandb/settings
2025-03-29 17:20:16,021 INFO    MainThread:42864 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-29 17:20:16,021 INFO    MainThread:42864 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /Users/liuyihao/Desktop/m2cw/wandb/run-20250329_172016-oc0lu6ll/logs/debug.log
2025-03-29 17:20:16,021 INFO    MainThread:42864 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /Users/liuyihao/Desktop/m2cw/wandb/run-20250329_172016-oc0lu6ll/logs/debug-internal.log
2025-03-29 17:20:16,021 INFO    MainThread:42864 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x382a554f0>
2025-03-29 17:20:16,022 INFO    MainThread:42864 [wandb_init.py:init():761] calling init triggers
2025-03-29 17:20:16,022 INFO    MainThread:42864 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-29 17:20:16,022 INFO    MainThread:42864 [wandb_init.py:init():784] starting backend
2025-03-29 17:20:16,022 INFO    MainThread:42864 [wandb_init.py:init():788] sending inform_init request
2025-03-29 17:20:16,050 INFO    MainThread:42864 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-03-29 17:20:16,050 INFO    MainThread:42864 [wandb_init.py:init():798] backend started and connected
2025-03-29 17:20:16,053 INFO    MainThread:42864 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-29 17:20:16,054 INFO    MainThread:42864 [wandb_init.py:init():891] updated telemetry
2025-03-29 17:20:16,084 INFO    MainThread:42864 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-29 17:20:16,598 INFO    MainThread:42864 [wandb_init.py:init():990] starting run threads in backend
2025-03-29 17:20:16,654 INFO    MainThread:42864 [wandb_run.py:_console_start():2375] atexit reg
2025-03-29 17:20:16,654 INFO    MainThread:42864 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-29 17:20:16,654 INFO    MainThread:42864 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-29 17:20:16,654 INFO    MainThread:42864 [wandb_run.py:_redirect():2315] Redirects installed.
2025-03-29 17:20:16,656 INFO    MainThread:42864 [wandb_init.py:init():1032] run started, returning control to user process
2025-03-29 17:20:16,657 INFO    MainThread:42864 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:20:16,657 INFO    MainThread:42864 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:20:16,660 INFO    MainThread:42864 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:20:50,069 INFO    MainThread:42864 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:20:50,069 INFO    MainThread:42864 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:20:50,074 INFO    MainThread:42864 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:20:50,076 INFO    MainThread:42864 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:20:50,076 INFO    MainThread:42864 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:20:50,078 INFO    MainThread:42864 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:20:50,079 INFO    MainThread:42864 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:20:50,079 INFO    MainThread:42864 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:20:50,081 INFO    MainThread:42864 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:20:50,083 INFO    MainThread:42864 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:20:50,083 INFO    MainThread:42864 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:20:50,085 INFO    MainThread:42864 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:20:50,933 INFO    MainThread:42864 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:20:50,934 INFO    MainThread:42864 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:20:50,937 INFO    MainThread:42864 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:20:52,573 INFO    MainThread:42864 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:20:52,574 INFO    MainThread:42864 [wandb_init.py:_pause_backend():564] pausing backend
2025-03-29 17:20:52,580 INFO    MainThread:42864 [wandb_init.py:_resume_backend():569] resuming backend
2025-03-29 17:20:52,594 INFO    MainThread:42864 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /Users/liuyihao/Desktop/m2cw/wandb/run-20250329_172052-vw28j5e1/logs/debug.log
2025-03-29 17:20:52,594 INFO    MainThread:42864 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /Users/liuyihao/Desktop/m2cw/wandb/run-20250329_172052-vw28j5e1/logs/debug-internal.log
2025-03-29 17:20:52,594 INFO    MainThread:42864 [wandb_init.py:init():761] calling init triggers
2025-03-29 17:20:52,594 INFO    MainThread:42864 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-29 17:20:52,594 INFO    MainThread:42864 [wandb_init.py:init():773] finishing previous run: oc0lu6ll
2025-03-29 17:20:52,594 INFO    MainThread:42864 [wandb_run.py:_finish():2112] finishing run liuyihao649-university-of-cambridge/Qwen_baseline/oc0lu6ll
2025-03-29 17:20:52,594 ERROR   MainThread:42864 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-29 17:20:52,594 INFO    MainThread:42864 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-29 17:20:52,594 INFO    MainThread:42864 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-29 17:20:52,594 INFO    MainThread:42864 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 0
2025-03-29 17:20:52,595 INFO    MainThread:42864 [wandb_run.py:_restore():2322] restore
2025-03-29 17:20:52,595 INFO    MainThread:42864 [wandb_run.py:_restore():2328] restore done
2025-03-29 17:20:53,342 INFO    MainThread:42864 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-29 17:20:53,342 INFO    MainThread:42864 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-29 17:20:53,343 INFO    MainThread:42864 [wandb_run.py:_footer_sync_info():3917] logging synced files
